content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -193,6 +193,84 @@ it does the following when a Pod is created:
 1. If the spec of the incoming Pod doesn't already contain any `imagePullSecrets`, then the
    admission controller adds `imagePullSecrets`, copying them from the `ServiceAccount`.
 
+### Legacy ServiceAccount token cleaner
+
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+
+Starting from Kubernetes version 1.28, the feature `LegacyServiceAccountTokenTracking`
+is enabled by default, which creates a ConfigMap that records the configmap
+creation date as the tracking date. With the feature enabled, each use of the
+legacy secret token updates the secret label `kubernetes.io/legacy-token-last-used`,
+with the latest usage date as the value.
+
+Starting from Kubernetes version 1.29, the legacy ServiceAccount token cleaner
+is enabled by default. It runs as part of `kube-controller-manager`
+and acts asynchronously. Every 24 hours, the cleaner checks if the current time
+has been a `speficied time` since the ConfigMap tracking date. If true, it loops
+the secret list and evaluates the secrets that have the type `kubernetes.io/service-account-token`.
```

Comment:
You mean "it identifies Secrets about ServiceAccount tokens" ...
I'd suggest we focus on the things users need to know. It is okay to explain how things work, but still, please focus on the implications to users.

For example, if the ConfigMap is kind of implementation details users don't need
to know, we can skip that. If the users may need to know it because something
bad would happen and they can mitigate those situations by manipulating this
ConfigMap, we can elaborate on that.

Reply:
Thank you, Qiming, explained above. Please refer this doc for the whole design if you are interested: https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2799-reduction-of-secret-based-service-account-token#design-details